src/master-playlist-controller.js

```diff
@@ -85,16 +79,15 @@ export default class MasterPlaylistController extends videojs.EventTarget {
 
       // if this isn't a live video and preload permits, start
       // downloading segments
-      if (media.endList &&
-          this.tech_.preload() !== 'metadata' &&
-          this.tech_.preload() !== 'none') {
+      if (media.endList && this.tech_.preload() !== 'none') {
         this.mainSegmentLoader_.playlist(media);
         this.mainSegmentLoader_.expired(this.masterPlaylistLoader_.expired_);
         this.mainSegmentLoader_.load();
       }
 
       this.setupSourceBuffer_();
       this.setupFirstPlay();
+      this.useAudio();
       this.trigger('loadedmetadata');
```

Comment:
Not a good reason, but I kept it around so that I could listen to it for one of the tests in `videojs-contrib-hls.test.js`. We can probably remove it